records/hic_hal/stm32h743ii.yaml

```diff
@@ -18,14 +18,12 @@ common:
         - source/hic_hal/stm32/stm32h743ii/STM32H7xx_HAL_Driver
         - source/hic_hal/stm32/stm32h743ii/STM32H7xx_HAL_Driver/Inc
         - source/hic_hal/stm32/stm32h743ii/STCubeGenerated/Inc
-        - source/hic_hal/stm32/stm32h743ii/extended_features/include
```

Comment:
Deletion of this line would make `main_interface.c` fail to include `i2c.h`. My opinion would be keeping this one and remove the line `source/hic_hal/stm32/stm32h743ii/extended_features/include` in `records/board/stm32h743ii_if.yaml`

Reply:
Actually the issue is deeper than that...

Reply:
I am not quite sure how the build file should be look like. I just proposed a workaround that would make both interface and bootloader build locally.